pkg/epinio/edit/applications.vue

```diff
@@ -69,8 +69,10 @@ export default Vue.extend<Data, any, any, any>({
         );
 
         await this.value.forceFetch();
-        saveCb(true);
-        this.done();
+        if (!this._isBeingDestroyed || !this._isDestroyed) {
```

Comment:
Comment only. At first i thought this should be &&, but `_isBeingDestroyed` is true even when `_isDestroyed` is true.